src/test/java/org/broadinstitute/hellbender/tools/PrintReadsIntegrationTest.java

```diff
@@ -156,7 +156,29 @@ public void testNoConflictPG() throws IOException {
         //output has both GATK PrintReads and GATK PrintReads.1 in headers
         Assert.assertNotNull(SamReaderFactory.makeDefault().open(outFile).getFileHeader().getProgramRecord("GATK PrintReads"));
         Assert.assertNotNull(SamReaderFactory.makeDefault().open(outFile).getFileHeader().getProgramRecord("GATK PrintReads.1"));
+    }
+
+    @Test
+    public void testPGOnByDefault() throws IOException {
+        final File inFile = new File(TEST_DATA_DIR, "print_reads_withPG.sam");
+        final File outFile = BaseTest.createTempFile("testNoConflictRG", ".sam");
+        final String[] args = new String[] {
+                //NOTE: no addOutputSAMProgramRecord argument
+                "--input" , inFile.getAbsolutePath(),
+                "--output", outFile.getAbsolutePath()
+        };
+        runCommandLine(args);
 
+        //Make sure contents are the same
+        SamAssertionUtils.assertSamsEqual(outFile, inFile);
```

Comment:
Does it matter whether the contents are the same for the purposes of this test? We don't want this test to fail unless there's an actual problem with the `PG` line itself.